Digital Resume Project

ygakkhar-tv opened the pull request.

ygakkhar-tv committed:
responsive changes

ygakkhar-tv committed:
change

ygakkhar-tv committed:
Merge branch 'feature/utkarsh-project' of github.com:ygakkhar-tv/web-project-resume into feature/digital-resume

# Conflicts:
#	index.html
#	style.css
#	template1.css

ygakkhar-tv committed:
final code sumbit

ygakkhar-tv committed:
code review 1

Mr-pm-tv commented on classes.js:
```diff
@@ -0,0 +1,32 @@
+
+class Skills{
+  constructor(s_kill){
+    this.s_kill = s_kill;
```
rename this variable as discussed

Mr-pm-tv commented on classes.js:
```diff
@@ -0,0 +1,32 @@
+
+class Certificate{
+  constructor(CField, CAuth, CDate){
+    this.CField = CField;
```
rename these variable as discussed

Mr-pm-tv commented on classes.js:
```diff
@@ -0,0 +1,32 @@
+}
+
+class Experience{
+  constructor(sDate, tDate, cName){
```
rename these variable as discussed

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+let table_body = document.getElementById("table-body");
+
+education_btn.addEventListener("click", () => {
+  let tr = document.createElement("tr");
```
rename this variable as discussed

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+education_btn.addEventListener("click", () => {
+  let tr = document.createElement("tr");
+
+  let template = `
```
rename this variable. we can rename this to educationTemplete

ksr877 replied:
Please use a dynamic way to add table rows, js have insertRow method to add new row

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+
+//skills add and clear button 
+let length=1;
+let skill_count = 1001;
```
remove underscore from variable

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+//skills add and clear button 
+let length=1;
+let skill_count = 1001;
+let skill_btn = document.getElementById("add_skills");
```
remove underscore from variable

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+experience_btn.addEventListener("click", () => {
+  let experienceDiv = document.createElement("div");
+  flag = flag + 1;
+  let template = `
```
rename this variable to "experienceTeamplete"

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+        <input type="text" name="Certification_feild" id="Certification_feild" class="certificate-field-class"/>
+        <input type="text" name="Certification_authority" id="Certification_authority" class="certificate-auth-class"/>
+        <button id="${count}" onclick="removeCertificate(this)">Clear</button>
+    `;
```
move this semicolon in line 105

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+
+
+//generate button
+let bool=true;
```
change this variable name 
For example, if I'm creating a variable to perform the check for the button so I can declare a variable with the name: "isButtonVisible"

ksr877 commented on index.js:
```diff
@@ -0,0 +1,171 @@
+//education add and clear button 
```
Please follow the camel case for variable name

Mr-pm-tv commented on localStorage.js:
```diff
@@ -0,0 +1,143 @@
+//skills
+let noOfSkills = length;
+
+console.log("no Of skills: " +noOfSkills);
```
remove log's from final code base

Mr-pm-tv commented on localStorage.js:
```diff
@@ -0,0 +1,143 @@
+    experienceArr[i] = new Experience("", "", "");
+  }
+
+  let eIndex = 0;
```
rename this variable to a relevint name

ksr877 replied:
Please use menaingful name

ygakkhar-tv committed:
FORM CSS UPDATION

Mr-pm-tv commented on index.js:
```diff
@@ -0,0 +1,171 @@
+//generate button
+let bool=true;
+
+let fname_state = true;
```
variable name!

ksr877 commented on localStorage.js:
```diff
@@ -0,0 +1,143 @@
+
+  //personal info
+  const Fname = document.getElementById('f-name').value;
+  localStorage.setItem("FName", Fname);
```
Please try to set the values in the localStorage using localStorage.setItem('formData',JSON.stringify({fn: firstName.value,
                    ln: lastName.value,
                    emailAddress: emailAddress.value,
                    phone: phone.value,
                    linkedin: linkedin.value,
                    github: github.value,
                    address: address.value,
                    aboutME: aboutME.value
                }));
Only for example

ksr877 commented on localStorage.js:
```diff
@@ -0,0 +1,143 @@
+for (let i = 0; i <noOfSkills; i++) {
+  skillArr[i] = new Skills("");
+}
+let sIndex = 0;
```
Use meaningful name for variable

ksr877 commented on style.css:
```diff
@@ -0,0 +1,164 @@
+
+textarea {
+  width: 95%;
+  padding: 2%;
```
Please always try px insisted of %.

ygakkhar-tv replied:
tried pixels but % is working much better

ksr877 commented on style.css:
```diff
@@ -0,0 +1,164 @@
+  border: none;
+}
+#generate-btn {
+  margin-bottom: 2%;
```
Please always try px insisted of %.

ksr877 commented on template1.css:
```diff
@@ -0,0 +1,144 @@
+.main {
+  display: grid;
+  grid-template-columns: 44% 1% 55%;
+  padding-left: 2%;
```
Please always try px insisted of %.

ygakkhar-tv replied:
tried pixels but % is working much better

ksr877 commented on template1.css:
```diff
@@ -0,0 +1,144 @@
+}
+
+.edu-2 {
+  padding: 10px 0px 0px 0px;
```
We can use these if you want to apply only one and the other is 0px
padding-top
padding-right
padding-bottom
padding-left

ksr877 commented on template1.css:
```diff
@@ -0,0 +1,144 @@
+  border-radius: 5px;
+  border: none;
+  font-weight: bold;
+  width: 10%;
```
Please always try px insisted of %.

ksr877 commented on template2.css:
```diff
@@ -0,0 +1,98 @@
+* {
```
margin and padding always default set 0px

ygakkhar-tv replied:
default not zero

ksr877 commented on template2.html:
```diff
@@ -0,0 +1,67 @@
+            <p id="aboutme"></p>
+          </div>
+          <hr />
+          <div class="skills margin" id="skill-data">
```
skills margin is not valid name pls use valid naming property

ygakkhar-tv replied:
multiple classes added here

ksr877 commented on template2.html:
```diff
@@ -0,0 +1,67 @@
+          <hr />
+          <div class="exeperince margin">
+            <h2>Certificates</h2>
+            <div class="box flex" id="certificate-data"></div>
```
Please use hyphen between class name

ygakkhar-tv replied:
multiple classes are used here

ksr877 commented on template2.html:
```diff
@@ -0,0 +1,67 @@
+          <hr />
+          <div class="work-experience" id="experience-details">
+            <h2>Work Experience</h2>
+            <div class="box flex" id="experience-data"></div>
```
Please use hyphen between class name

ksr877 commented on template2.html:
```diff
@@ -0,0 +1,67 @@
+      </div>
+    </div>
+
+    <div class="templ1">
```
use meaningful name

ksr877 commented on validations.js:
```diff
@@ -0,0 +1,242 @@
+  var FName = /^[a-zA-Z]+[a-zA-Z]+$/;
+  if (!f_name.match(FName) || f_name.length < 4) {
+    focus_func(document.getElementById("f-name"));
+    bool = false;
```
use meaningful name for variable

niteshktv committed:
changes
-skill div bottom gap reduce
-education heading font-size
-choose template font size
-generate button move upward

ygakkhar-tv committed:
css changed

ygakkhar-tv committed:
Merge branch 'feature/nitesh' of github.com:ygakkhar-tv/web-project-resume into feature/digital-resume

# Conflicts:
#	index.js
#	style.css

ygakkhar-tv committed:
placeholder changed

niteshktv committed:
Merge branch 'feature/utkarsh-project' of https://github.com/ygakkhar-tv/web-project-resume into feature/nitesh

niteshktv committed:
commited from utkarsh

niteshktv committed:
remove unwanted files during merge

niteshktv committed:
merged with utkarsh regarding validation code

Mr-pm-tv reviewed (changes requested):
modify required changes ASAP

niteshktv committed:
Captailize Placeholders

ygakkhar-tv committed:
code review changes
